framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -1676,6 +1965,77 @@ protected final void setLintOption(String name, boolean val) {
     }
   }
 
+  /**
+   * Returns the set of subchecker classes on which this checker depends. Returns an empty set if
+   * this checker does not depend on any others.
+   *
+   * <p>Subclasses should override this method to specify subcheckers. If they do so, they should
+   * call the super implementation of this method and add dependencies to the returned set so that
+   * checkers required for reflection resolution are included if reflection resolution is requested.
+   *
+   * <p>If you wish to create a checker that runs multiple checkers, but itself is not a type
```

Comment:
Is this line about running multiple independent checkers, or multiple checkers that may depend on one another?